### ISSUE TITLE
json-only mode for frodo info, don't print the other messages

### ISSUE THREAD

**Issue by lohman** [rockcarver/frodo#199]
Hi there,

When I run a frodo info, it prints out 5 messages before it gives me the json.  It would be nice if there was a switch that I could give to frodo info to have it avoid echoing anything to console other than the json.

**Comment by atomicsamurai** [comment]
`frodo info` has a script friendly switch, does that not help? For example, in my case, it prints the first couple of lines on STDERR, but dumps the tokens etc. on STDOUT, which I pipe to `jq` and subsequently another script.
```
frodo info -s sandeep | jq -r .bearerToken | jwt
ForgeRock Identity Cloud detected.
Connected to ForgeRock Access Management 7.2.0-2022-4-SNAPSHOT Build b107f663c5a014de548ca0f17b333c682d244f48 (2022-May-19 13:40)
{
    "sub": "fc8ad8da-af68-49c7-aba1-2df2aa383e84",
    "cts": "OAUTH2_STATELESS_GRANT",
    "auth_level": 0,
    "auditTrackingId": "8f0c3da9-7742-41e8-92d8-6f29232a4ae8-848922",
    "subname": "fc8ad8da-af68-49c7-aba1-2df2aa383e84",
    "iss": "https://openam-sandeep-dev.forgeblocks.com:443/am/oauth2",
    "tokenName": "access_token",
    "token_type": "Bearer",
    "authGrantId": "lDy8nBVO-btlaExKoww_QvnUn2c",
    "aud": "idmAdminClient",
    "nbf": "2022-06-08T15:12:52Z",
    "grant_type": "authorization_code",
    "scope": [
        "openid",
        "fr:idm:*"
    ],
    "auth_time": 1654701171,
    "realm": "/",
    "exp": "2022-06-08T16:12:52Z",
    "iat": "2022-06-08T15:12:52Z",
    "expires_in": 3600,
    "jti": "BLO3fePm0I6C6snztQ819pYbZg4"
}
```

**Comment by lohman** [comment]
Yes, you're right, I should have mentioned that.  My feature request was lazy. I apologize for that.

The -s does help, yes, thank you.  Even with that, I'm finding I prefer to suppress stderr with something like `frodo info https://dmdcbenefits.catalyst.forgerock.org/am forgerock 123456 -s dmdc 2> /dev/null` so that I only get pure json data in the console.  I find the suppression of stderr makes it easier for me to work with the output.  This feature request therefore is simply a matter of personal preference and convenience.  Yes, technically, I can still pipe it in to awk or jq or whatever, and I only get the json.  The implementation is technically sound.  It's just that my eyeballs prefer to have a built-in option to only see the stdout so that I don't have to work around it with the `2> /dev/null`.

**Comment by atomicsamurai** [comment]
Ok, we can leave this one open for now as low priority and revisit later. At least it works for you functionally.
